README.md

```diff
@@ -7,7 +7,10 @@
 This project is a library that is able to ingest [EVM](https://ethereum.org/en/developers/docs/evm/)
 [bytecode](https://ethereum.org/en/developers/docs/evm/opcodes/) and discover an approximate storage
 layout for the contract described by that bytecode. It is _not_ intended to be a full decompiler,
-but is instead a tool **highly** specialised for performing this discovery process.
+but is instead a tool **highly** specialised for performing this discovery process. 
+
+See our [announcement](https://blog.smlxl.io/a-deep-dive-into-our-storage-layout-extractor-51554185d8af) for more details
```

Comment:
@dorlevi  both links are to the deepdive post - shouldn't the first one be to this https://blog.smlxl.io/announcing-bytecode-generated-storage-layouts-on-evm-storage-96761758d397 ?

Reply:
🤦